README.md

```diff
@@ -233,8 +233,8 @@ end of the path part of URL (preceding the query).
 The session is stored as a UUID inside a cookie.
 The cookie name is one of the following by descending order of priority:
 
-* Using the `com.amadeus.session.cookieName` initialization parameter of the ServletContext.
```

Comment:
Would be good to also specify in the previous section (Session propagation) how to change the propagation:

As per standard via web.xml
<web-app> <session-config> <tracking-mode>URL</tracking-mode>

Or via web app init params com.amadeus.session.tracking

And also specify the version limitation as detailed in Usage.md